developing.adoc

```diff
@@ -46,40 +46,40 @@ For more information, see the following:
 . link:https://github.com/golang/go/wiki/Modules[Introduction to Go modules]
 . link:https://blog.golang.org/using-go-modules[Using Go modules]
 
-[[running-integration-tests]]
-== Running integration tests
+[[running-e2e-tests]]
+== Running e2e tests
 
-We have automated integration tests which keep CodeReady Containers in shape.
+We have automated e2e tests which keep CodeReady Containers in shape.
 
-[[intro-to-integration-testing]]
+[[intro-to-e2e-testing]]
 === Introduction
-Integration tests use the link:http://github.com/code-ready/clicumber[Clicumber] package to provide basic functionality for testing CLI binaries.
+E2e tests use the link:http://github.com/code-ready/clicumber[Clicumber] package to provide basic functionality for testing CLI binaries.
```

Comment:
Same comment about the e2e acronym. And it looks odd written `E2e` rather than `e2e`